### PR TITLE
Feature/add datepicker component

### PR DESCRIPTION
**Pull request by jasperswart**
Screenshot 2021-04-28 at 19 00 44

- [x] You've added or updated the README.mdx of the story of the component
- [x] You have been following the guidelines, written in the [README](../blob/master/docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [x] Your code has the necessary tests written
- [x] You've exported the component in [index.ts](./packages/asc-ui/src/index.ts)
- [x] You have updated the [CHANGELOG.md unreleased sections](../blob/master/CHANGELOG.md#muser-content-unreleased)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "test:watch": "jest --watch --onlyChanged --verbose true"
   },
   "devDependencies": {
+    "@amsterdam/asc-assets": "^0.29.3",
```
Comment (jonkoops) [inline package.json:28]: This dependency should not be added here. The packages should be linked by NPM automatically.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by jonkoops** [comment]
One small thing, otherwise LGTM

File: package.json (hunk 1 above)
Comment (jasperswart) [reply package.json:28]: weg

Comment (jasperswart) [reply package.json]: weg

**Pull request merged**